src/components/clusterexplorer/explorer.ts

```diff
@@ -92,6 +96,17 @@ export class KubernetesExplorer implements vscode.TreeDataProvider<ClusterExplor
         });
     }
 
+    initialize() {
+        const viewer = vscode.window.createTreeView('extension.vsKubernetesExplorer', {
+            treeDataProvider: this
+        });
+		return vscode.Disposable.from(
```

Comment:
Please convert tabs to spaces - it's throwing off the indentation. Thanks!

Reply:
It's hard for me to fix this kind of indentation issue bc in my ide everything is written correctly. Wouldn't it be better to add a script to fix this kind of issues automatically? Like eslint or something similar?